packages/web-components/src/components/dotcom-shell/__stories__/dotcom-shell.stories.ts

```diff
@@ -80,9 +80,9 @@ const StoryContent = () => html`
 `;
 
 export const Default = ({ parameters }) => {
-  const { brandName, userStatus, navLinks } = parameters?.props?.['dds-masthead-container'] ?? {};
+  const { brandName, userStatus, navLinks } = parameters?.props?.Masthead ?? {};
   const { langDisplay, language, size: footerSize, legalLinks, links: footerLinks, localeList } =
```

Comment:
For fixing Prettier error:

```suggestion
  const { langDisplay, language, size: footerSize, legalLinks, links: footerLinks, localeList } = parameters?.props?.Footer ?? {};
```